### PR TITLE
Add Dockerfile and its instructions

### PR DESCRIPTION
**Pull request by jadia**
This PR adds support for a third way to run the program. README.md file is updated to show instructions on how to use docker to run this script.
The size of the Docker image created with this Dockerfile is ~85MB.

### CODE REVIEW

**Comment by TheWeirdDev** [comment]
Why would anyone need this though? Running a container for a simple python script seems a bit too much.

**Comment by jadia** [comment]
> Why would anyone need this though? 

I run almost everything on containers. I created a Dockerfile for myself out of need, so created a PR for the same. In case someone else needs a container for this. No issues if this is unnecessary for this repository.

**Comment by TheWeirdDev** [comment]
I'll merge this in case someone finds it useful. But first I need to change some stuff.

File: Readme.md [hunk 1]
```diff
@@ -41,6 +41,28 @@ _make sure you have `python-pybluez` or `python3-pybluez` or `python3-bluez` ins
...
 
+#### 3. Docker
+
+You must have docker installed on your system.
+Use the below command to install or refer to [this guide](https://docs.docker.com/engine/install/).
+
+```bash
+curl -fsSL get.docker.com | sudo bash
+```
+
```
Comment (TheWeirdDev) [inline Readme.md:52]: The info on docker installation is unnecessary
```suggestion
```

File: Readme.md [hunk 2]
```diff
@@ -41,6 +41,28 @@ _make sure you have `python-pybluez` or `python3-pybluez` or `python3-bluez` ins
...
+```bash
+cd Bluetooth_Headset_Battery_Level && \
+docker build -t bluetooth_battery_level .
+```
+
+Once the build is complete you can use the below command to run the program:
+
+```bash
+docker run --rm -ti --privileged --net=host bluetooth_battery_level "BT_MAC_ADDRESS"
+#docker run --rm -ti --privileged --net=host bluetooth_battery_level "E2:41:23:5C:4F:27"
```
Comment (TheWeirdDev) [inline Readme.md:64]:
```suggestion
```

**Review by TheWeirdDev — COMMENTED** [review]
Good enough

**Comment by jadia** [comment]
Great! 👍🏼

**Pull request merged**